src/features/utils/aggregate-base.js

```diff
@@ -0,0 +1,16 @@
+import { registerHandler } from '../../common/event-emitter/register-handler'
+import { FeatureBase } from './feature-base'
+
+export class AggregateBase extends FeatureBase {
```

Comment:
This class was originally removed because it didn't actually do anything. Nice to see it coming back with functionality. Are all the feature aggregate classes updated to extend this?